assets/components/LogViewer/CopyLogMessage.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div
+    class="absolute -right-1 flex min-w-[0.98rem] items-start justify-end align-bottom hover:cursor-pointer"
```

Comment:
Right now this component can't be used in non absolute style because styles are hard coded. Instead of having the styles here, remove the ones that are not relevant to the component to where you use it. eg `<copy-log-message :message="..." class="absolute -right-1.../>`. You can probably keep `items-start justify-end...` since those are needed.